src/components/manager.ts

```diff
@@ -478,17 +492,21 @@ export class Manager {
         return includedTeX
     }
 
-    private getDirtyContent(file: string, reload: boolean = false): string {
-        for (const cachedFile of Object.keys(this.cachedContent)) {
-            if (reload) {
-                break
-            }
-            if (path.relative(cachedFile, file) !== '') {
-                continue
+    /**
+     * Get the buffer content of a file if it is opened in vscode. Otherwise, read the file from disk
+     */
+    getDirtyContent(file: string): string | undefined {
+        const cache = this.cachedContent[file]
+        if (cache !== undefined) {
+            if (cache.content) {
+                return cache.content
             }
-            return this.cachedContent[cachedFile].content
         }
-        const fileContent = utils.stripCommentsAndVerbatim(fs.readFileSync(file).toString())
+        if (!fs.existsSync(file)) {
+            this.extension.logger.showErrorMessage(`Cannot read dirty content of ${file}`)
```

Comment:
We should call `this.extension.logger.addLogMessage` to log errors too.

Reply:
Do we really need to show a popup error with `showErrorMessage`? Maybe logging the error is enough.

Reply:
I agree that logging the error is enough.